openwisp_controller/config/settings.py

```diff
@@ -67,3 +67,6 @@ def get_setting(option, default):
     "API_TASK_RETRY_OPTIONS",
     dict(max_retries=5, retry_backoff=True, retry_backoff_max=600, retry_jitter=True),
 )
+GEOIP_ACCOUNT_ID = get_setting("GEOIP_ACCOUNT_ID", None)
+GEOIP_LICENSE_KEY = get_setting("GEOIP_LICENSE_KEY", None)
+WHO_IS_ENABLED = get_setting("WHO_IS_ENABLED", False)
```

Comment:
if this is t true, we need to assert that GEOIP_ACCOUNT_ID and GEOIP_LICENSE_KEY are set (not falsy) otherwise we should raise `ImproperlyConfigured`.